plugin/pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -130,7 +138,7 @@ func TestPodFitsResources(t *testing.T) {
 	}
 
 	for _, test := range enoughPodsTests {
-		node := api.Node{Status: api.NodeStatus{Capacity: makeResources(10, 20, 32).Capacity}}
+		node := api.Node{Status: api.NodeStatus{Capacity: makeResources(10, 20, 32).Capacity, Allocatable: makeAllocatableResources(10, 20, 32)}}
```

Comment:
Aren't you essentially changing the node-status updates here?  Why would you breakout the Allocatable, separate from Capacity?  IMHO we should create structure under capacity.  I must admit I didn't fully review this until now.  /cc @davidopp